jobs/rocoto/atmanalprep.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -x
+
+###############################################################
+# Source FV3GFS workflow modules
+. $HOMEgfs/ush/load_fv3gfs_modules.sh
+status=$?
+[[ $status -ne 0 ]] && exit $status
+
+###############################################################
+# set JJOB variables
+export configs="base atmanal atmanalprep"
+export EXSCRIPT=${GDASPREPPY:-$HOMEgfs/sorc/gdas.cd/scripts/exgdas_global_atmos_analysis_prep.py}
```

Comment:
These variables should not be in the rocoto job script. 
Thinking ahead for the `ecflow` equivalent, would they be in the `.ecf`?
`base` adn `atmanal` are common to the 3 `atmanalXXXX.sh` jobs.  The last one `prep` or `post` or `run` need to be passed through.
I can think on this a bit more, but wanted to put this down as a comment.

Reply:
You are correct.  This approach does not work with `ecflow`.    

Currently the GDASApp has a single j-job, `JGDAS_GLOBAL_ATMOS_ANALYSIS`, for atmospheric JEDI-VAR DA.   This single j-job is used for prep, run, and post with toggles placed in rocoto jobs.   

One way to refactor this is to have separate GDASApp j-jobs.  That is, we replace `JGDAS_GLOBAL_ATMOS_ANALYSIS` with

- `JGDAS_GLOBAL_ATMOS_ANALYSIS_PREP`
- `JGDAS_GLOBAL_ATMOS_ANALYSIS_RUN`
- `JGDAS_GLOBAL_ATMOS_ANALYSIS_POST`

The toggles currently in `atmanalprep.sh`, `atmanalrun.sh`, and `atmanalpost.sh` would then move into their respective j-jobs.
   
This conversation also applies to JEDI-ENS DA.   The GDASApp currently has a single j-job for JEDI-ENS DA prep, run, and post.   It, too, could be refactored as suggested above.

Thoughts, comments? @aerorahul , @CoryMartin-NOAA , @WalterKolczynski-NOAA

Reply:
[6674720](https://github.com/NOAA-EMC/global-workflow/commit/667472055b8db74e05467f0c7b02eb6a67bce9d1) split the multifunction `JGDAS_GLOBAL_ATMOS_ANALYSIS` and `JGDAS_GLOBAL_ATMOS_ENSANAL` j-jobs into function specific j-jobs as outlined above.